tests/stdlib/trst.nim

```diff
@@ -358,7 +406,7 @@ OtherStart
 .. include:: other.rst
              :start-after: OtherStart
 """
-    doAssert "<em>Visible</em>" == rstTohtml(input, {}, defaultConfig())
+    check "<em>Visible</em> " == rstTohtml(input, {}, defaultConfig())
```

Comment:
how about using strbasics.strip (inline strip) at the end of `rstTohtml` instead?

Reply:
good idea. Applied it.